section4/exercises/classMethods.js

```diff
@@ -28,13 +28,56 @@ Make sure to run the file with node in your command line.
 // that the proteins have been changed.
 
 class Burrito {
-  constructor(protein, base) {
+  constructor(protein, base, fillers) {
     this.protein = protein;
     this.base = base;
+      // ADD CODE
+    this.topping = fillers;
   }
+addTopping() {
+  console.log("Add Cilantro");
```

Comment:
HOw could you write this so that it updates your data model? Ideally utilizing a parameter so that it's dynamic